### PR TITLE
Spanish translation of the file philosophy.mdx

### PR DESCRIPTION
**Pull request by jkmg**
# What does this PR do?

Fixes # (issue)

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [x] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [x] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

### CODE REVIEW

**Comment by jkmg** [comment]
Spanish translation of the file philosophy.mdx #15947

**Comment by omarespejel** [comment]
Thank you @jkmg! Could you please add `philosophy` to `../transformers/docs/source/es/_toctree.yml`. According to the [new Translation](https://github.com/huggingface/transformers/blob/main/docs/TRANSLATING.md) guide (section "✍️ Start translating")? This would allow the tests to pass.

File: docs/source/es/philosophy.mdx [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+Unos cuantos objetivos adicionales:
+
+- Exponer las características internas de los modelos de la forma más coherente posible:
+
+  - Damos acceso, mediante una sola API, a todos los estados ocultos y pesos de atención.
+  - Tokenizer y el modelo de API base están estandarizados para cambiar fácilmente entre modelos.
+
+- Incorporar una selección subjetiva de herramientas de gran potencial para la optimización/investigación de estos modelos:
+
+  - Una forma sencilla/coherente de añadir nuevos tokens al vocabulario e incrustraciones ("embeddings") para optimización.
```
Comment (omarespejel) [inline docs/source/es/philosophy.mdx:52]:
```suggestion
  - Una forma sencilla/coherente de añadir nuevos tokens al vocabulario e incrustraciones (embeddings, en inglés) para optimización.
```

**Review by omarespejel — CHANGES_REQUESTED** [review]

**Review by jkmg — COMMENTED** [review]
Is it now OK?

**Review by omarespejel — APPROVED** [review]

**Pull request merged**